climada/util/test/test_coordinates.py

```diff
@@ -230,6 +236,47 @@ def def_ref_50():
     )
 
 
+class TestGetUnitCoords(unittest.TestCase):
+    def create_mock_gdf(self, crs, num_points=5):
+        geometry = [
+            shapely.Point(x, y) for x, y in zip(range(num_points), range(num_points))
+        ]
+        gdf = gpd.GeoDataFrame(geometry=geometry, crs=crs)
+        return gdf
```

Comment:
This is called a *test fixture*. Please use `setUp` to create fixtures and save them in the `TestCase` object. You can then retrieve them from the `self` object within each test case. See https://docs.python.org/3/library/unittest.html#organizing-test-code

Reply:
Indeed, I was wrong about the usage of a fixture here. What you do is more of a [parameterized test](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html#parametrize-basics), which unittest cannot really facilitate 🤷